src/index.ts

```diff
@@ -17,6 +17,10 @@ export interface HtmlFileConfiguration {
     scriptLoading?: 'blocking' | 'defer' | 'module',
     favicon?: string,
     findRelatedOutputFiles?: boolean,
+    extraScripts?: (string | { 
```

Comment:
I think we want to call this `attrs` instead of `tags`.
Are there any particular attributes that are relevant here for you? Maybe it makes sense to ask for the fields individually.

Reply:
Renamed to attrs in latest commit. I just need async for mine but figured I'd leave it open for others to customize. If you want to add props for each available attribute though, I can add it.